typo on index.d.ts types getting an error on typescript compiler

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Explain what you did

Compile the project with tsc --noEmit --skipLibCheck and get the following error: 

```bash
> yarn tsc --noEmit --skipLibCheck 
yarn run v1.22.15
$ /Users/rafo/code/repositories/rocket-bank-mobile/node_modules/.bin/tsc --noEmit --skipLibCheck
node_modules/@rneui/themed/dist/config/index.d.ts:2:15 - error TS1005: ',' expected.

2 import { type Colors, darkColors, lightColors } from './colors';
                ~~~~~~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```







### Expected behavior

No errors when running `tsc` cli command 

### Describe the bug

Typescript compiling typo error on `themed/src/config/index.d.ts`

### Steps To Reproduce

```Markdown
1. go to themed/src/config/index.d.ts with typescript enabled
```


### Screenshots

<img width="413" alt="Screen Shot 2022-07-17 at 18 00 57" src="https://user-images.githubusercontent.com/53544360/179424582-ad61fc9c-bfdb-4956-9954-0a09f163150f.png">


<img width="1440" alt="Screen Shot 2022-07-17 at 17 54 58" src="https://user-images.githubusercontent.com/53544360/179424609-8f7d70bd-5ef1-4cdb-ba42-551fe3b997bc.png">

### Your Environment

<details>
No results, just indefinitely locking terminal process. 

package.json

    "rneui": "^1.0.0-stable.0",
    "expo": "~45.0.0",
    "@types/react": "~17.0.21",
    "@types/react-native": "^0.69.3",
    "@typescript-eslint/eslint-plugin": "^5.30.6",
    "@typescript-eslint/parser": "^5.30.6",
    "eslint": "^8.19.0",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-react": "^7.30.1",
    "eslint-plugin-react-hooks": "^4.6.0",
    "typescript": "~4.3.5"
    "resolutions": {
      "@types/react": "17.0.44"
    },


<summary>`npx @rneui/envinfo`</summary>

      ```
        Output from `npx @rneui/envinfo` goes here.
      ```

</details>

--- a/packages/themed/src/config/index.ts
+++ b/packages/themed/src/config/index.ts
@@ -13,7 +13,7 @@ import {
   ScreenWidth,
   registerCustomIconType,
 } from '@rneui/base/dist/helpers';
-import type { Colors, darkColors, lightColors } from './colors';
+import { Colors, darkColors, lightColors } from './colors';
 import {
   ReplaceTheme,
   UpdateTheme,

--- a/packages/themed/src/config/index.ts
+++ b/packages/themed/src/config/index.ts
@@ -13,7 +13,7 @@ import {
   ScreenWidth,
   registerCustomIconType,
 } from '@rneui/base/dist/helpers';
-import { type Colors, darkColors, lightColors } from './colors';
+import type { Colors, darkColors, lightColors } from './colors';
 import {
   ReplaceTheme,
   UpdateTheme,

